rpcs3/Emu/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-﻿add_library(rpcs3_emu
+add_library(rpcs3_emu
```

Comment:
was this in BOM format or why did this change?

Reply:
It was in BOM